.travis.yml

```diff
@@ -47,7 +29,7 @@ deploy:
   provider: pypi
   user: Claudiu.Popa
   password:
-    secure: YElO/mU+n8JcHdOkbD7Q14XqQce7ydu9/zbm6gpqEOvI/CMXyjsBaWKTg9LFAm8HtRMsaRfa4FElmFOXmY4LPqkRAIKNvxOzWnD76lsPlrIpHF7oHlxPQcLAO0YXcrCCwQh5NYm06KX8n5Wv2ypHqfDJv8QuPSl8v+2PCDwx2jeCLeo8GfuAmGJWxNn7IIAmAD3U0Gyc1FZ2KGtKcS9mNoLYRO9zZykomOVfhgQjZw6x7NJTg8x7vm5QOEe1nwoc8/5m7brI7ZeF+eFFaXrQOu+OMRSJnt8W0dr4mgNa71CEDVBAJxqQzy8EkkMaonOCvpiJckUvXfy+ovdiDpL8r1X+GnQJ4fK838tOM7BAIIBzIwfNzWrPXNMGJtD2Ws9zZr/FAqFpjHo2dSavcvqjknm1kO6OmpLB4fYlW+HS0pcS2hINuMof20jR74WlrrXgj3uQIOtSI94Y8ipREDv+TRAT1G82h6qkT1vgf27ksYumXZwTIi191YR3cmqM3xD/z1j1ZrWbsxTJMzA4Ia+qcQN1cInn8bFGrU8agHrzufcfeT+t7cvTlRhAF90JCq4ViycvCad8qDu5+0C3XpqppE6naC2yWFd08EQ1xPyFBLDyohZdEdIN7Ob9Dm6ZYy8YGcx04sl7fmAEIf20HM9pDTdPODfSnMZOiMa5/fLgcKk=
+    secure: lAlz/mySOEOqIMp9vYb6WVvd4YP/XmnP1XmDJWAziit4+ydSB52H0wUprBZjMHenChtflANIKXggiaVO6sw6EqU8mxMEMz+6ixs9ZA0robYy9CgYdMrXSAYgr8NHbf3WPTiD65ajP5bpQ/v6i5YhVXhTgotORBmhnMyn5LA/OvbQGWZqHsdtdXZpsflXuzEDD9SL/MgrvfOEBINJzHuXyKDqwOzqjNL9VeUoUHbubBk/haJtbXHPvAQR9SOtS1hBeq9sVAQghdxQTs39XNPAnzukgEwW0UNmmuW6bQ6UWbxztHHQYgXBni5cfhGE7B5GO2L0Cneuiwz99HGyDvdOSNgxNahLcIlAWCWzp71T7KSRnPhAFMVbw7/65eb5VIJKyrO9rwZi5zCo4+c9Wi0er7+l1PVLcEw9O+ouEYs1+1iY7JFyP4cHAPGd6h0POG/IE3UJZ/5yhOSBR6sYwRbR4Qc2zPflnZrjSgBCpaJ37Y+FZwg7BzPvElGteTmqm3PsdqWWJshYs/l5QaRuzUOalPlxJHDrau9JPm3KAlosJde7cUD5zooiy08GHfd8fle2zAbGjgk9p7VAFf/2BFJj261h9eAmFHwIgBW7jje3eBCYUbBuzl+uzGGQNdfyoNzrbRcnuVWr/Is9PefVf0OmLDPNTgJy0gevsMZgfoCCuiQ=
```

Comment:
Are you sure this should be changed?

Reply:
The deployment on tag is not working for astroid, but its working for pylint. This is a bet where I lose nothing if it does not work 😄

Reply:
Didn't know that. So you uploaded all releases to pypi manually in the past?

Reply:
Only for astroid :)

Reply:
https://github.com/PyCQA/astroid/blob/master/doc/release.md